app/screens/sso/sso.js

```diff
@@ -99,7 +99,7 @@ class SSO extends PureComponent {
     }
 
     clearPreviousCookies = () => {
-        CookieManager.clearAll(true).then(() => {
+        CookieManager.clearAll(false).then(() => {
```

Comment:
Thanks @pesintta for your contribution, the problems that I see here are:
1. You'll need to set the WebView to not to use WebKit as well
2. The use of UIWebView has been deprecated, that is why we are using WKWebView instead.

Reply:
Hi. Thank you for the comments.
As to the first observation the following change can be done (though I'm not entirely certain that this would really be needed - WKWebView and WKHTTPCookieStore were introduced in different iOS versions):
```suggestion
@@ -241,7 +241,7 @@ class SSO extends PureComponent {
                     injectedJavaScript={jsCode}
                     onLoadEnd={this.onLoadEnd}
                     onMessage={messagingEnabled && this.onMessage}
-                    useWebKit={true}
+                    useWebKit={false}
                 />
             );
         }
```

Unfortunately I've got no idea how to properly resolve the second comment.
App Store allows mattermost-mobile to be installed to devices with iOS 9.3 or later so it can be installed to devices which do not have WKHTTPCookieStore (but do have WKWebView) and therefore are prone to the bug this PR is trying to resolve.

Reply:
@pesintta I think the best approach here is to use WebKit for OS 11+ and to fallback to the previous UIWebView on devices with the OS lower than 11. On the other hand we may very well bump the required OS version to be 11+